packages/react-liveness/src/components/FaceLivenessDetector/service/utils/StreamRecorder/StreamRecorder.ts

```diff
@@ -123,6 +130,43 @@ export class StreamRecorder {
     });
   }
 
+  /**
+   * The startFaceLivenessSession API takes in a ReadableStream as its source of all websocket events
+   * To allow for changing cameras we add new MediaRecorders which pass through events to the previously set MediaRecorder
+   * @param oldRecorder
+   */
+  #createPassThroughRecorder(): void {
+    this.#recorder.ondataavailable = ({ data }) => {
+      this.#initialRecorder.dispatchEvent(
+        new MessageEvent('dataavailable', { data })
+      );
+    };
+
+    this.#recorder.addEventListener('sessionInfo', (e: any) => {
```

Comment:
Do we need some sort of cleanup to remove the event listeners if they switch to a different recorder?

Reply:
added event listener clean up when they switch recorders